Tests/SwiftLintFrameworkTests/ExplicitInitRuleTests.swift

```diff
@@ -13,10 +13,24 @@ class ExplicitInitRuleTests: SwiftLintTestCase {
             Example("foo(↓.init())")
         ]
 
+        var correction = [
+            Example("""
+            f { e in
+                A↓.init(e: e)
+            }
+            """):
+                Example("""
+                f { e in
+                    A(e: e)
+                }
+                """)
+        ]
+        correction.merge(ExplicitInitRule.description.corrections) { current, _ in current }
```

Comment:
The example can be added directly to `ExplicitInitRule.description.corrections`. No need to update it here.